### ISSUE TITLE
Part List

### ISSUE THREAD

**Issue by gadjodilo83** [BasicCode/Toshiba-T1200-PSU#1]
Hi!

A great project! 
Is there a simple parts list of the components?

You may be able to provide us with a confirmation (e.g. Digikey) of your order for the components. 
I have already ordered the PCB, thank you for your efforts!

Thanks
Marc

**Comment by BasicCode** [comment]
Hi,
Glad you're finding it useful. Sorry for the delay with the parts list. Most of it came from Digikey but a few parts were just general items I had laying around in boxes so I had to try and find them on Digikey. I have made a list which contains MOST of the parts needed.
https://github.com/BasicCode/Toshiba-T1200-PSU/blob/3f3613d4c2f3fc39c498c8727aeb8e1fbedf5eb0/Toshiba%20T1200%20Power%20Supply.xlsx

A couple of parts are missing because I couldn't find them:

- The connectors for the cables linking the power supply to the main board. I found the headers but not the female cable sockets. I got mine from Jaycar:
https://www.jaycar.com.au/4-pin-0-1-header-with-crimp-pins-2-54-pitch/p/HM3404
https://www.jaycar.com.au/2-pin-0-1-header-with-crimp-pins-2-54-pitch/p/HM3402

- I used generic resistors and ceramic caps, however the 33uF electrolytic should be low ESR caps from a reputable brand.
- Use an LED colour of your choice. :)
